app/uk/gov/hmrc/agentservicesaccount/controllers/SignOutController.scala

```diff
@@ -17,16 +17,26 @@
 package uk.gov.hmrc.agentservicesaccount.controllers
 
 import javax.inject.Inject
-
+import play.api.Configuration
+import play.api.i18n.{I18nSupport, MessagesApi}
 import play.api.mvc.{Action, AnyContent}
 import uk.gov.hmrc.agentservicesaccount.config.ExternalUrls
+import uk.gov.hmrc.agentservicesaccount.views.html.signed_out
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 
 import scala.concurrent.Future
 
-class SignOutController @Inject()(externalUrls: ExternalUrls) extends FrontendController {
+class SignOutController @Inject()(implicit val externalUrls: ExternalUrls, configuration: Configuration, val messagesApi: MessagesApi)
+  extends FrontendController with I18nSupport {
 
-  val signOut: Action[AnyContent] = Action.async { implicit request =>
+  def signOut(timeout: Option[String]): Action[AnyContent] = Action.async { implicit request =>
```

Comment:
is this method only ever being called with Some("t") as the parameter from the timeout dialog? Confused. Will always go to the sign_out screen...

Reply:
If the user clicks the sign out link does it still go to the sign_out screen?

Reply:
But it's always being passed Some("t"). So when you click the sign out link on the timeout pop up it will show the timeout page which says something like 'you have been inactive for 15 mins....bla bla' which doesn't make sense because they clicked a link to sign out